packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -81,24 +81,24 @@ const ButtonStyled = styled(Button)<{ theme: ResolvedTheme }>`
     width: 100%;
 
     &[aria-expanded='true'] {
-        background: ${({ theme }) => theme.greys.white};
+        background: ${({ theme }) => theme.component['accordion-header-extended-background-color']};
         border-radius: var(--border-radius-2x) var(--border-radius-2x) 0 0;
-        color: ${({ theme }) => theme.greys['neutral-90']};
+        color: ${({ theme }) => theme.component['accordion-header-extended-color']};
         transition-delay: 0s;
         transition-duration: 0.1s;
         transition-property: border-radius;
         transition-timing-function: ease;
     }
 
     &:focus {
-        box-shadow: ${({ theme }) => theme.tokens['focus-box-shadow-inset']};
-        color: ${({ theme }) => theme.greys['neutral-90']};
+        box-shadow: inset 0 0 0 2px ${({ theme }) => theme.component['accordion-header-focus-box-shadow-inset-color']};
```

Comment:
Plus bas (e.g., dans la Modal), tu n'utilises pas `box` dans le naming. Le seul autre shadow qui serait possible, est un `text-shadow` et je ne sais pas si nous allons en avoir un moment donné.
@savutsang @meriouma @maboilard Vous en pensez quoi? est-ce que on laisse ou non le `box`?
@ogermain-kronos Uniformiser avec le reste des tokens plus bas quand nous aurons une décision de prise.

Reply:
EDIT
Henri utilise le `box` dans ses tokens donc nous allons le garder ici aussi pour uniformiser tout ça. Faire la modification dans la modal par contre.

Reply:
En ce qui a trait à la présence du `box` dans le nom, j'avais juste pris le nom que j'ai trouvé dans les legacy tokens. J'ai pas d'opinion sur ce qui pourrait constituer un meilleur nom (ou pas). Je vais vous laisser argumenter.

Mais sinon ouais t'as raison, j'ai oublié le `box` dans un des tokens de la modale.

Reply:
On va ajouter le `box` il est pas mal partout dans les autres PR des tokens feck ca va être plus simple l'ajouter ici que de refaire une passe dans toutes les autres PR.

Reply:
Yup. Je l'avais inclu dans mon dernier commit.

Reply:
I thank you